analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -58,13 +58,7 @@ public sealed class SymbolicExecutionRunner : SymbolicExecutionRunnerBase
         protected override ImmutableDictionary<DiagnosticDescriptor, RuleFactory> AllRules => ImmutableDictionary<DiagnosticDescriptor, RuleFactory>.Empty
             .Add(LocksReleasedAllPaths.S2222, CreateFactory<LocksReleasedAllPaths>());
 
-        public SymbolicExecutionRunner()
-        {
-            foreach (var descriptor in SonarRules.SelectMany(x => x.SupportedDiagnostics))
-            {
-                RegisterSupportedDiagnostics(descriptor);
-            }
-        }
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => base.SupportedDiagnostics.Concat(SonarRules.SelectMany(x => x.SupportedDiagnostics)).ToImmutableArray();
```

Comment:
The expression body should go on the next line.

Reply:
It's the same as in this thread https://github.com/SonarSource/sonar-dotnet/pull/5393#discussion_r806808351

>The current formatting was intentional as we're sure that the list of .Add lines will grow (each on its own line). And the Empty part is just a boring scaffolding noise.